### PR TITLE
user plugin with 0.12.x changes

### PR DESCRIPTION
**Pull request by ralphiee22**
Test/compare with https://instantschools-v2.learningequality.org.

### CODE REVIEW

File: kolibri_instant_schools_plugin/kolibri_plugin.py [hunk 1]
```diff
@@ -21,18 +25,111 @@ class UserInclusionHook(hooks.UserSyncHook):
...
-        return about_urls
+class LogInRedirect(RoleBasedRedirectHook):
+    role = ANONYMOUS
+
+    @property
+    def url(self):
+        return self.plugin_url(User, "user")
+
+
+#  Navigation hooks
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/kolibri_plugin.py:36]: Now using pluggable side navigation hooks.

File: kolibri_instant_schools_plugin/kolibri_plugin.py [hunk 2]
```diff
@@ -21,18 +25,111 @@ class UserInclusionHook(hooks.UserSyncHook):
...
 class AboutAsset(webpack_hooks.WebpackBundleHook):
     unique_slug = "about_module"
-    src_file = "assets/src/views/about/aboutApp.js"
+    src_file = "assets/src/AboutApp.js"
 
 
 class AboutInclusionHook(hooks.AboutSyncHook):
     bundle_class = AboutAsset
+
+
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/kolibri_plugin.py:64]: Using theme hook plugin.

File: kolibri_instant_schools_plugin/assets/src/views/SignInPage/FacilityModal.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<template>
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/assets/src/views/SignInPage/FacilityModal.vue:1]: This file may not be necessary for VF case.

File: kolibri_instant_schools_plugin/assets/src/AboutApp.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import router from 'kolibri.coreVue.router';
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/assets/src/AboutApp.js:1]: About/FAQ page is its own module.

File: kolibri_instant_schools_plugin/assets/src/views/SideNav/AboutSideNavEntry.vue [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      CoreMenuOption,
+    },
+    $trs: {
+      about: 'About',
+    },
+    computed: {
+      url() {
+        return urls['kolibri:about:about']();
+      },
+    },
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/assets/src/views/SideNav/AboutSideNavEntry.vue:35]: no role is specified to always show `about` tab in navigation

File: kolibri_instant_schools_plugin/assets/src/views/SignUpPage.vue [hunk 6]
```diff
@@ -34,161 +19,168 @@
...
-        <label for="terms-agreement-checkbox">
-          <button
-            type="button"
-            class="terms-agreement-view-prompt"
-            @click="showTerms = true"
-          >
-            {{ $tr('viewTermsOfServicePrompt') }}
-          </button>
-        </label>
+      <template v-if="currentFacility">
```
Comment (ralphiee22) [inline kolibri_instant_schools_plugin/assets/src/views/SignUpPage.vue:61]: do we even want to show the current/only facility for VF?

**Review by ralphiee22 — COMMENTED** [review]
No changes were made to the backend API. Tried keeping things in the UI consistent, such as leaving any styling that was already there from the user plugin. Updated the way frontend api calls were made by keeping the calls local to the component they are in.

**Comment by indirectlylit** [comment]
Seeing a few issues - not sure how many are due to local config:

image

image

image

image

**Pull request merged**